content/repositories/working-with-files/using-files/viewing-a-file.md

```diff
@@ -1,21 +1,34 @@
 ---
-title: Tracking changes in a file
-intro: You can trace changes to lines in a file and discover how parts of the file evolved over time.
+title: Viewing a file
+intro: You can view raw file content or trace changes to lines in a file and discover how parts of the file evolved over time.
 redirect_from:
   - /articles/using-git-blame-to-trace-changes-in-a-file
   - /articles/tracing-changes-in-a-file
   - /articles/tracking-changes-in-a-file
   - /github/managing-files-in-a-repository/tracking-changes-in-a-file
   - /github/managing-files-in-a-repository/managing-files-on-github/tracking-changes-in-a-file
+  - /repositories/working-with-files/using-files/tracking-changes-in-a-file
 versions:
   fpt: '*'
   ghes: '*'
   ghae: '*'
   ghec: '*'
 topics:
   - Repositories
-shortTitle: Track file changes
+shortTitle: View files and track file changes
 ---
+## Viewing or copying the raw file content
```

Comment:
Since viewing a raw file and right-clicking to save as doesn't really download the file, but rather a `.txt` file copy of the content, I've changed this from "downloading" to "copying." Copying the raw content is supported in the GitHub user interface, so we should definitely document it here with the other aspects of viewing a raw file.

Downloading a single file isn't the intended use of git or GitHub, so we don't want to document a workaround for that.